src/app/routes/getInitialData/index.js

```diff
@@ -6,6 +6,8 @@ import onClient from '#lib/utilities/onClient';
 import getBaseUrl from './utils/getBaseUrl';
 import getPreprocessorRules from './utils/getPreprocessorRules';
 
+const qs = require('querystringify');
```

Comment:
```suggestion
import qs from 'querystringify';
```

Reply:
`querystringify` no longer required, but thanks for this